src/plugins/discover/public/application/view_components/utils/use_search.ts

```diff
@@ -220,10 +222,7 @@ export const useSearch = (services: DiscoverViewServices) => {
 
     let elapsedMs;
     try {
-      // Only show loading indicator if we are fetching when the rows are empty
-      if (fetchStateRef.current.rows?.length === 0) {
-        data$.next({ status: ResultStatus.LOADING, queryStatus: { startTime } });
-      }
+      data$.next({ status: ResultStatus.LOADING, queryStatus: { startTime: currentTime } });
```

Comment:
So we wish to always to show loading indicator no matter if the rows are empty or not?

Reply:
yes